src/pages/components/ProductSummary.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react'
+import { Box, Composition } from 'atomic-layout'
+import styled from 'styled-components'
+import { Image } from '../../atoms/Image'
+import { Heading } from '../../atoms/Heading'
+import { Button } from '../../atoms/Button'
+import { FaFacebookF, FaTwitter, FaInstagram } from 'react-icons/fa'
+import { Rating } from '../../molecules/Rating'
+
+const templateMobile = `
+	gallery
+	summary
+`
+
+const templateDesktop = `
+	gallery summary
+	/ 1fr 1fr
+`
+
+const StyledDescription = styled.p`
+  color: #999;
+  line-height: 28px;
+`
+
+const StyledProdactPrice = styled.span`
+  color: ${({ theme }) => theme.colors.greyLight};
+  font-size: 1.7rem; /* 24px */
```

Comment:
You can also write your helper function that would accept pixels but return rems:

```js
// Declaration
export const rem = (pixels: number): string => {
  const rems = pixels / 16 // this assumes you have "16px" as 1rem
  return `${rems}rem`
}

// Usage
import { rem } from '../somewhere'

const Styled = styled.span`
  font-size: ${rem(24)}; // you ask yo convert "24px" to rems
`
```

Reply:
does it mean it must be rewritten all rem to this function which changes to the px?

Reply:
Not necessarily. Just suggesting you a useful method you can use :) Decide on your own if you want to use it or not. It doesn't block this pull request.